packages/integration-sdk-runtime/src/execution/__tests__/config.test.ts

```diff
@@ -13,32 +13,38 @@ jest.mock('fs');
 beforeEach(() => {
   process.env.STRING_VARIABLE = 'string';
   process.env.BOOLEAN_VARIABLE = 'true';
+  process.env.STRING_ARRAY_VARIABLE = ' string1, string2 ';
 });
 
 afterEach(() => {
   delete process.env.STRING_VARIABLE;
   delete process.env.BOOLEAN_VARIABLE;
+  delete process.env.STRING_ARRAY_VARIABLE;
 
   vol.reset();
 });
 
 test('loads config fields from environment variables', () => {
   const instanceConfigFields: IntegrationInstanceConfigFieldMap<
-    Record<'stringVariable' | 'booleanVariable', IntegrationInstanceConfigField>
+    Record<'stringVariable' | 'booleanVariable' | 'stringArrayVariable', IntegrationInstanceConfigField>
   > = {
     stringVariable: {
       type: 'string',
     },
     booleanVariable: {
       type: 'boolean',
     },
+    stringArrayVariable: {
+      type: 'string[]',
```

Comment:
I also considered `array`, which is consistent with what's in `web-frontend`. But wonder if in the future we might want to load arrays of numbers, etc as well.